### ISSUE TITLE
[new feature] Adding a summary report

### ISSUE THREAD

**Issue by pachevalier** [dataproofer/Dataproofer#122]
Dataproofer seems to be a very interesting tool. However, it would be nice to export a small report. It could be a small markdown file. 

This would be useful to send the report to the data producer. For instance, people could look at datasets on data.gouv.fr such as [this one](https://www.data.gouv.fr/fr/datasets/dgddi-nomenclature-combinee-a-8-chiffres-nc8/) and send a report to the organization to improve the formatting of the spreadsheet.

This could also be a useful tool for organizations which release open datasets.

**Comment by ejfox** [comment]
@pachevalier Agreed! We've been talking about this a bit.

Would you be able to paste what you think the markdown should look like?

Something like:

```
Missing rows: 20%, 5 rows
Duplicate rows: 0%, 0 rows
```

Or more detail? Or less?

**Comment by pachevalier** [comment]
Yeah this would be nice. It could also be something more structured such as a YAML or a JSON.

**Comment by geraldarthur** [comment]
@pachevalier just wanted to give you a quick update that I've been working on a JSON output from the CLI. i'm currently working on the `cli` branch if you'd like to help us kick the tires on this a little bit sometime soon
